siliconcompiler/templates/tcl/manifest.tcl.j2

```diff
@@ -12,3 +21,9 @@ proc sc_top {} {
     }
     return $sc_entrypoint
 }
+
+{% if scroot  %}
```

Comment:
Is there any scenario where we wouldn't want to have this?
Just asking because if it's conditional then we will always have to check if it's present when using it later.